actionpack/lib/action_controller/metal/implicit_render.rb

```diff
@@ -2,8 +2,14 @@ module ActionController
   module ImplicitRender
     def send_action(method, *args)
       ret = super
-      default_render unless performed?
-      ret
+      return ret if performed?
+
+      if template_exists?(method, _prefixes, variants: request.variant || [])
```

Comment:
I could wrap it in `Array`. `request.variant` returns nil without variants, which errors further down the line in a .compact. Doing `Array(request.variant` would prevent that, or make it so request always returns an array